index.bs

```diff
@@ -423,17 +421,18 @@ The following additional constraints apply only to [=AV1 Image Sequences=]:
 - The AV1 level for High Profile shall be 5.1 or lower.
 
 <div class="example">
-A file containing items compliant with this profile is expected to list the following brands, in any order, in the [=compatible_brands=] of the [=FileTypeBox=]:
+    A file containing items compliant with this profile is expected to list the following brands, in any order, in the [=compatible_brands=] of the [=FileTypeBox=]:
 
-  <code>avif, mif1, miaf, MA1A</code>
+    <code>avif, mif1, miaf, MA1A</code>
 
-A file containing a 'pict' track compliant with this profile is expected to list the following brands, in any order, in the [=compatible_brands=] of the [=FileTypeBox=]:
+    A file containing a 'pict' track compliant with this profile is expected to list the following brands, in any order, in the [=compatible_brands=] of the [=FileTypeBox=]:
 
-  <code>avis, msf1, miaf, MA1A</code>
+    <code>avis, msf1, miaf, MA1A</code>
 </div>
 
-  <h2 id="mime-registration">AVIF Media Type Registration</h2>
+<h2 id="mime-registration">AVIF Media Type Registration</h2>
 
 <p>The media type <code>"image/avif"</code> is officially registered with IANA and available at: <a href="https://www.iana.org/assignments/media-types/image/avif">https://www.iana.org/assignments/media-types/image/avif</a>.</p>
 
-  <h2 id="change-list">Changes since v1.1.0 release</h2>
+<h2 id="change-list">Changes since v1.1.0 release</h2>
```

Comment:
Is this a list of changes in the source or in the generated document?

What does this PR do to the generated document?

Reply:
Ah. That is a good question. The generated document ends up _looking_ almost the same, but some of the defined terms in the document metadata are changed. So the Webref scraper will see different stuff for example.

The main noticeable difference for human readers is that `AV1 Item Configuration Property` is now `AV1ItemConfigurationProperty`.

Reply:
I guess we need to figure out what the policy is regarding which changes end up in this list.

Reply:
You could list every PR and mark these kinds of PR as editorial.

Reply:
> You could list every PR and mark these kinds of PR as editorial.

I ended up doing this.